core/collector.class.inc.php

```diff
@@ -594,7 +594,7 @@ protected function AddRow($aRow)
 	{
 		$aData = array();
 		foreach ($this->aCSVHeaders as $sHeader) {
-			if (is_null($aRow[$sHeader]) && $this->AttributeIsNullified($sHeader)) {
+			if (strlen($aRow[$sHeader])==0 && $this->AttributeIsNullified($sHeader)) {
```

Comment:
```suggestion
			if (strlen($aRow[$sHeader] ?? '') === 0 && $this->AttributeIsNullified($sHeader)) {
```

Otherwise in case a null value (as it seem it can be possible) it will trigger a warning with PHP 8.

Reply:
thank you. correction is pushed like recommended